tsconfig.example.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./tsconfig.json",
+  "extends": ["./renovate.json", "./tsconfig.json"],
```

Comment:
that's not what I meant :) 
Remove all changes from this file, and then add the `extends` thing to `renovate.json`